test/relations.test.js

```diff
@@ -2776,10 +2953,14 @@ describe('relations', function() {
     it('should find record on scope with promises', function(done) {
       Passport.findOne()
       .then(function(p) {
-        p.personId.should.eql(personCreated.id);
+        if (!isCassandraConnector) {
```

Comment:
I find it suspicious that ` it('should find record on scope'` can assert `p.personId` is `personCreated.id` but the promise variant cannot. What's the reason behind that?

Can we use `Person.findById(personCreated.id)` instead of `Person.findOne` to avoid this problem?